agent/__init__.py

```diff
@@ -129,21 +135,20 @@ def is_certificate_expired():
     return datetime.datetime.now(datetime.timezone.utc) > get_certificate_expiration_date()
 
 
-def generate_device_id(debug=False):
+def generate_device_id():
     """
     Device ID is generated remotely.
     """
     device_id_request = requests.get(
         '{}/v0.2/generate-id'.format(WOTT_ENDPOINT)
     ).json()
 
-    if debug:
-        print("[RECEIVED] Generate Device ID: {}".format(device_id_request))
+    logging.debug("[RECEIVED] Generate Device ID: {}".format(device_id_request))
```

Comment:
why do you do `logging.debug()` instead of using previously created logger?
also fix this in the `agent/executor.py` file

Reply:
fixed